cli/Valet/Site.php

```diff
@@ -464,10 +464,13 @@ public function secured()
      *
      * @param  string  $url
      * @param  string  $siteConf  pregenerated Nginx config file contents
-     * @param  int  $caExpireInDays  The mount of days the self signed certificate is valid.
+     * @param  int  $certificateExpireInDays  The amount of days the self signed certificate is valid.
+     *                                        Certificates SHOULD NOT have a validity period greater than 397 days.
+     * @param  int  $caExpireInYears  The amount of years the certificate authority is valid.
+     * @see https://github.com/cabforum/servercert/blob/main/docs/BR.md
      * @return void
      */
-    public function secure($url, $siteConf = null, $caExpireInDays = 368)
+    public function secure($url, $siteConf = null, $certificateExpireInDays = 396, $caExpireInYears = 20)
```

Comment:
I'm not sure why `368` was picked by @icyrizard, according to the baseline-requirements it can be 396 days:

https://github.com/cabforum/servercert/blob/main/docs/BR.md (2020-09-01)

Reply:
Yah, I checked the original PR: https://github.com/laravel/valet/pull/1154 

And there's no explanation given, so I'm totally fine for moving it to 396. Good catch!